.changeset/shaggy-hornets-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': major
```

Comment:
I believe this is not major change? We are adding attribute, not changing current API.

Reply:
Updated